### PR TITLE
Cookies also have a version number.

### PR DESCRIPTION
**Pull request by victornoel**
Hi,

Pac4j implementation of cookies was missing the version field.

### CODE REVIEW

**Comment by leleuj** [comment]
+1

This change, of course, does not bring anything new until the specific web contexts implement it.

**Comment by victornoel** [comment]
Ah yes, you are right, I forgot to do so ^^ I will add what is needed!

**Comment by victornoel** [comment]
@leleuj here it is for the pac4j repository.

File: pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java [hunk 1]
```diff
@@ -202,6 +202,7 @@ public String getFullRequestURL() {
         if (cookies != null) {
             for (final javax.servlet.http.Cookie c : cookies) {
                 final Cookie cookie = new Cookie(c.getName(), c.getValue());
+                cookie.setVersion(c.getVersion());
```
Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java]: Is version 0 the default value? Just to be sure we don't break anything for those who use cookies.

File: pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java [hunk 2]
```diff
@@ -217,6 +218,7 @@ public String getFullRequestURL() {
     @Override
     public void addResponseCookie(final Cookie cookie) {
         final javax.servlet.http.Cookie c = new javax.servlet.http.Cookie(cookie.getName(), cookie.getValue());
+        c.setVersion(cookie.getVersion());
```
Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java]: The question fits here best: is version 0 the default value? Just to be sure we don't break anything for those who use cookies.

Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java]: Perfect

**Pull request merged**